server/tests/src/test/java/org/infinispan/server/persistence/PooledConnectionOperations.java

```diff
@@ -112,6 +91,65 @@ public void testPutGetRemove() {
       assertEquals("v2", cache.get("k2"));
       cache.remove("k1");
       assertNull(cache.get("k1"));
+      assertCleanCacheAndStore(cache);
+   }
+
+   @Test
+   public void testPreload() throws Exception {
+      JdbcConfigurationUtil jdbcUtil = new JdbcConfigurationUtil(CacheMode.REPL_SYNC, database, false, true)
+              .setLockingConfigurations();
+      RemoteCache<String, String> cache = SERVER_TEST.hotrod().withServerConfiguration(jdbcUtil.getConfigurationBuilder()).create();
+      cache.put("k1", "v1");
+      cache.put("k2", "v2");
+
+      SERVERS.getServerDriver().stop(0);
+      SERVERS.getServerDriver().restart(0);
+
+      // test preload==true, entries should be immediately in the cache after restart
+      assertEquals(2, cache.size());
+      assertEquals("v1", cache.get("k1"));
+      assertEquals("v2", cache.get("k2"));
+      cache.clear();
+   }
+
+   /*
+    * This should verify that DefaultTwoWayKey2StringMapper on server side can work with ByteArrayKey which
+    * is always produced by HotRod client regardless of type of key being stored in a cache.
+    */
+   @Test
+   public void testDefaultTwoWayKey2StringMapper() throws Exception {
+      JdbcConfigurationUtil jdbcUtil = new JdbcConfigurationUtil(CacheMode.DIST_SYNC, database, false, false);
+      RemoteCache<Object, Object> cache = SERVER_TEST.hotrod().withServerConfiguration(jdbcUtil.getConfigurationBuilder()).withQualifier("1").create();
+
+      Double doubleKey = 10.0;
+      Double doubleValue = 20.0;
+      assertEquals(0, cache.size());
+      cache.put(doubleKey, doubleValue);
+      assertEquals(doubleValue, cache.get(doubleKey));
+   }
+
+   @Test
+   public void testSoftRestartWithPassivation() throws Exception {
+      JdbcConfigurationUtil jdbcUtil = new JdbcConfigurationUtil(CacheMode.REPL_SYNC, database, true, false)
+              .setEvition()
+              .setLockingConfigurations();
+      RemoteCache<String, String> cache = SERVER_TEST.hotrod().withServerConfiguration(jdbcUtil.getConfigurationBuilder()).create();
+      cache.put("k1", "v1");
+      cache.put("k2", "v2");
+      cache.put("k3", "v3");
+
+     //now k3 is evicted and stored in store
```

Comment:
Nit: Spacing got messed up for the rest of the file

Reply:
@wburns fixed